src/graphql/index.ts

```diff
@@ -19,12 +19,16 @@ const server = new ApolloServer({
   typeDefs,
   resolvers,
   tracing: true,
+  introspection: true,
   playground: {
     // @ts-ignore
     shareEnabled: true,
     tabs: [
       {
-        endpoint: 'https://score.snapshot.org/graphql',
+        endpoint:
```

Comment:
What if the endpoint is different here? Cant you get the current url?

Reply:
@bonustrack No way to get the current URL automatically. We have to define a URL for sure, or maybe we can add URL in`.env` and display that

Reply:
How about using `./graphql` ?

Reply:
well `/graphql` works but fails when we share playground 
https://www.graphqlbin.com/v2/vDlqhW

![image](https://user-images.githubusercontent.com/15967809/123053501-95f80200-d421-11eb-8c6f-e9e36c38933f.png)

Reply:
The share button is terrible, not sure why they send user to another site.

Reply:
Ya 😓 actually `shareEnabled` is not in official documentation too .. maybe they are just trying possibilities, what to do? 🙊  should commit with `/graphql`? I think the condition I added should be fine for now 😋

Reply:
The repo snapshot-score don't need GraphQL, and i'm not sure we want to change to Apollo for the hub, to me it just looks more pretty but it doesn't help anything.